### PR TITLE
Add pagetitle to request show page in webui2

### PR DESCRIPTION
**Pull request by krauselukas**
We always show a pagetitle

### CODE REVIEW

File: src/api/app/views/webui2/webui/request/show.html.haml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+:ruby
+  @pagetitle = "Show Request #{@number}"
+  @layouttype = 'custom'
```
Comment (DavidKang) [inline src/api/app/views/webui2/webui/request/show.html.haml]: Are we using this?

Comment (dmarcoux) [reply src/api/app/views/webui2/webui/request/show.html.haml]: No, it's a remnant from the [old webui](https://github.com/openSUSE/open-build-service/blob/5b51a39e1b448c2116c9e70715b2fe79593e041e/src/api/app/views/layouts/webui/webui.html.erb#L80)

We need to remove it in the other webui2 views. Only a few have this unneeded instance variable.

Comment (vpereira) [reply src/api/app/views/webui2/webui/request/show.html.haml]: I use it in the Overview. As it is part of the Overview, I suggest to close this PR..

Comment (dmarcoux) [reply src/api/app/views/webui2/webui/request/show.html.haml]: I will send a PR to remove the other webui2 views which have this unneeded instance variable.

Comment (dmarcoux) [reply src/api/app/views/webui2/webui/request/show.html.haml]: The PR: #6852

Comment (dmarcoux) [reply src/api/app/views/webui2/webui/request/show.html.haml]: @krauselukas we can close this PR.

**Pull request closed without merging**